apps/src/templates/rubrics/LearningGoals.jsx

```diff
@@ -124,8 +136,15 @@ export function annotateLines(observations) {
       for (let i = lineNumber; i <= lastLineNumber; i++) {
         EditorAnnotator.highlightLine(i);
       }
+      ret.push({
+        firstLine: lineNumber,
+        lastLine: lastLineNumber,
+        message: message,
+      });
     }
   }
+
+  return ret;
```

Comment:
Need to now return the lines that we annotated to write them out in the assessment box.